docs/source/community/mission-scope.md

```diff
@@ -3,25 +3,50 @@
+`movement` aims to **facilitate the study of animal behaviour**
+by providing a suite of **Python tools to analyse body movements**
+across space and time.
 
 ## Scope
 
-At its core, movement handles trajectories of *keypoints*, which are specific body parts of an *individual*. An individual's posture or *pose* is represented by a set of keypoint coordinates, given in 2D (x,y) or 3D (x,y,z). The sequential collection of poses over time forms *pose tracks*. In neuroscience, these tracks are typically extracted from video data using software like [DeepLabCut](dlc:) or [SLEAP](sleap:).
-
-With movement, our vision is to present a **consistent interface for pose tracks** and to **analyze them using modular and accessible tools**. We aim to accommodate data from a range of pose estimation packages, in **2D or 3D**, tracking **single or multiple individuals**. The focus will be on providing functionalities for data cleaning, visualisation and motion quantification (see the [Roadmap](target-roadmaps) for details).
-
-While movement is not designed for behaviour classification or action segmentation, it may extract features useful for these tasks. We are planning to develop separate packages for this purpose, which will be compatible with movement and the existing ecosystem of related tools.
+At its core, `movement` handles the positions of one or more individuals
+tracked over time. An individual's position at a given time can be represented
+in various ways: a single keypoint (usually the centroid), a set of keypoints
+(also known as the pose), a bounding box, or a segmentation mask.
+The spatial coordinates of these representations may be defined in 2D (x, y)
+or 3D (x, y, z). The pose and mask representations also carry some information
+about the individual's posture.
+
+Animal tracking frameworks such as [DeepLabCut](dlc:) or [SLEAP](sleap:) can
+generate these representations from video data by detecting body parts and
+tracking them across frames. In the context of `movement`, we refer to the
+resulting tracks according to their respective representations—for
+example, pose tracks, bounding boxes' tracks, or motion tracks in general.
+
+Our vision is to present a **consistent interface for motion tracks** paired
+with **modular and accessible analysis tools**. We aim to accommodate data
+from a range of animal tracking frameworks, in **2D or 3D**, tracking
+**single or multiple individuals**. As such, `movement` can be considered as
+downstream of tools like DeepLabCut and SLEAP. The focus is on providing
+functionalities for data cleaning, visualization, and motion quantification
+(see the [Roadmap](target-roadmaps) for details).
+
+In the study of animal behavior, motion tracks are often used to extract and
+label discrete actions, sometimes referred to as behavioral syllables or
+states. While `movement` is not designed for such tasks, it may generate
```

Comment:
```suggestion
states. While `movement` is not designed for such tasks, it can be used to generate
```

Reply:
"it can be used to generate features useful for" sounds awkard because of overusing "use" ;)

How about "While `movement` is not designed for such tasks, it may aid them by generating useful features and by inter-operating with existing action recognition tools".

Reply:
Sure, I forgot to replace "useful".

The sentence "it may aid them" sounds a bit cumbersome to me, but maybe we can check with a native what sounds more natural. Some other options
- "it can be used to generate features that may be relevant for action recognition" 
- "it can be used to support the generation of features that may be informative for action recognition"

Reply:
how about using the term "supplement"

Reply:
I like ""it can be used to generate features that may be relevant for action recognition".

Reply:
Thanks for the suggestions, both.